keepassxc-browser/popups/popup.css

```diff
@@ -109,6 +109,11 @@ body {
 	width: 100%;
 }
 
+#login-list {
```

Comment:
You cannot do this. This modifies the popup_login.html's login list to a scrollable list.

Reply:
Plus without this the dialog window itself is scrollable, not the list, which is much better even if the window doesn't scale dynamically.